package.json

```diff
@@ -175,6 +175,18 @@
         "command": "qute.validation.enabled.toggle.on",
         "title": "Enable Qute Validation",
         "icon": "$(eye-closed)"
+      },
+      {
+        "command": "qute.refactor.surround.with.comments",
```

Comment:
You can already surround with comment using `Ctrl+Shift+/`. Is it worthwhile having a separate command to do this/does it do something different in certain cases?

Reply:
The Qute comment uses the AST, so for instance if you wish comment full section, you set the cursor to the section and apply comment which will comment the full section content.

Reply:
Okay. That makes sense, and sound helpful.